builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -141,22 +148,85 @@ func (b *backend) pathUpdateIssuer(ctx context.Context, req *logical.Request, da
 		return nil, err
 	}
 
+	newName, err := getIssuerName(ctx, req.Storage, data)
+	if err != nil {
+		// If the error is name already in use, and the new name is the
+		// old name for this issuer, we're not actually updating the
+		// issuer name (or causing a conflict) -- so don't err out. Other
+		// errs should still be surfaced, however.
+		if !strings.Contains(err.Error(), "issuer name already in use") || newName != issuer.Name {
```

Comment:
Could we have a dedicated error for this use case? String comparison on the error msg feels a little fragile.